tests/lib/rules/require-explicit-slots.js

```diff
@@ -160,6 +160,21 @@ tester.run('require-explicit-slots', rule, {
           parser: null
         }
       }
+    },
+    // ignore attribute binding
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <div>
+          <slot :name="'foo'"></slot>
```

Comment:
Maybe we can add support for string literals, but ignore other attribute bindings? That shouldn't be too hard with the `isStringLiteral` and `getStringLiteralValue` util functions.

Reply:
Done ✅